test/webgpu_basic_rendering.cpp

```diff
@@ -400,7 +400,7 @@ void run() {
             .device = device,
             .format = capabilities.formats[0],
             .usage = wgpu::TextureUsage::RenderAttachment,
-            .alphaMode = wgpu::CompositeAlphaMode::Auto,
+            .alphaMode = wgpu::CompositeAlphaMode::Opaque,
```

Comment:
```suggestion
            .alphaMode = wgpu::CompositeAlphaMode::Auto,
```
(Or you could remove this and presentMode like you did in the sample.)